packages/react/src/analytics/integrations/Vitorino/providers/Riskified/Riskified.ts

```diff
@@ -0,0 +1,149 @@
+import {
+  EventData,
+  IntegrationFactory,
+  integrations,
+  LoadIntegrationEventData,
+  StrippedDownAnalytics,
+  TrackTypes,
+  TrackTypesValues,
+} from '@farfetch/blackout-analytics';
+import get from 'lodash/get';
+import type { RiskifiedIntegrationOptions } from './types';
+
+class Riskified extends integrations.Integration<RiskifiedIntegrationOptions> {
+  private isReady: boolean;
+  private siteId: string;
+  private baseUrl: string;
+
+  /**
+   * Creates an instance of Riskified and sets the script on the page.
+   *
+   * @param options - Custom options for the integration.
+   * @param loadData - Analytics's load event data.
+   */
+  constructor(
+    options: RiskifiedIntegrationOptions,
+    loadData: LoadIntegrationEventData,
+    analytics: StrippedDownAnalytics,
+  ) {
+    super(options, loadData, analytics);
+
+    this.isReady = false;
+    this.siteId = options.siteId || 'farfetch.com';
```

Comment:
See https://github.com/Farfetch/blackout/pull/649/files#r1092316527

Reply:
Update: It was removed the defaults, as we discuss offline.